docs/mint.json

```diff
@@ -368,51 +368,96 @@
     },
     {
-      "source": "/guides/streaming/destinations/kinesis"
+      "source": "/streaming/destinations/dummy",
+      "destination": "/guides/streaming/destinations/dummy"
     },
     {
-      "source": "/guides/streaming/destinations/mongodb"
+      "source": "/streaming/destinations/elasticsearch",
+      "destination": "/guides/streaming/destinations/elasticsearch"
     },
     {
-      "source": "/guides/streaming/destinations/opensearch"
+      "source": "/streaming/destinations/kafka",
+      "destination": "/guides/streaming/destinations/kafka"
     },
     {
-      "source": "/guides/streaming/destinations/redshift"
+      "source": "/streaming/destinations/kinesis",
+      "destination": "/guides/streaming/destinations/kinesis"
     },
     {
-      "source": "/guides/streaming/overview",
-      "destination": "/streaming/overview"
+      "source": "/streaming/destinations/mongodb",
+      "destination": "/guides/streaming/destinations/mongodb"
+    },
+    {
+      "source": "/streaming/destinations/mssql",
+      "destination": "/guides/streaming/destinations/mssql"
+    },
+    {
+      "source": "/streaming/destinations/mysql",
+      "destination": "/guides/streaming/destinations/mysql"
+    },
+    {
+      "source": "/streaming/destinations/opensearch",
+      "destination": "/guides/streaming/destinations/opensearch"
+    },
+    {
+      "source": "/streaming/destinations/postgres",
+      "destination": "/guides/streaming/destinations/postgres"
+    },
+    {
+      "source": "/streaming/destinations/redshift",
+      "destination": "/guides/streaming/destinations/redshift"
+    },
+    {
+      "source": "/streaming/destinations/snowflake",
+      "destination": "/guides/streaming/destinations/snowflake"
+    },
+    {
+      "source": "/streaming/destinations/trino",
+      "destination": "/guides/streaming/destinations/trino"
+    },
+    {
+      "source": "/streaming/sources/amazon-sqs",
+      "destination": "/guides/streaming/sources/amazon-sqs"
     },
     {
-      "source": "/guides/streaming/sources/amazon-sqs"
+      "source": "/streaming/sources/azure-event-hub",
+      "destination": "/guides/streaming/sources/azure-event-hub"
     },
     {
-      "source": "/guides/streaming/sources/azure-event-hub"
+      "source": "/streaming/sources/google-cloud-pubsub",
+      "destination": "/guides/streaming/sources/google-cloud-pubsub"
     },
     {
-      "source": "/guides/streaming/sources/google-cloud-pubsub"
+      "source": "/streaming/sources/kafka",
+      "destination": "/guides/streaming/sources/kafka"
     },
     {
-      "source": "/guides/streaming/sources/kafka"
+      "source": "/streaming/sources/kinesis",
+      "destination": "/guides/streaming/sources/kinesis"
     },
     {
-      "source": "/guides/streaming/sources/kinesis"
+      "source": "/guides/streaming-pipeline-rabbitmq",
+      "destination": "/guides/streaming/sources/rabbitmq"
```

Comment:
@colefreeman You should search for a doc named `streaming-pipeline-rabbitmq.mdx` instead. To keep the reference consistent, the destination should reference this doc: `docs/guides/streaming/tutorials/streaming-pipeline-rabbitmq.mdx`. We have an actual guide for rabbitmq specifically.